spec/unit/rom/gateway_spec.rb

```diff
@@ -87,4 +87,38 @@ class Gateway
       expect(gateway.disconnect).to be(nil)
     end
   end
+
+  describe '.adapter' do
+    let(:gateway_class) { Class.new(ROM::Gateway) }
+
+    it 'gets/sets the adapter' do
+      expect { gateway_class.adapter :custom }
+        .to change { gateway_class.adapter }
+        .from(nil)
+        .to(:custom)
+    end
+  end
+
+  describe "#adapter" do
+    subject { ROM::CustomAdapter.new.adapter }
```

Comment:
Ok, I'll correct this soon. (I just used to `subject` as "subject under test", but you're right - specs should be written in a consistent style)